python/src/com/jetbrains/python/debugger/containerview/NumericContainerViewTable.java

```diff
@@ -82,12 +82,15 @@ private void initUi(@NotNull final ArrayChunk chunk, final boolean inPlace) {
       myComponent.getSliceTextField().setText(chunk.getSlicePresentation());
       myComponent.getFormatTextField().setText(chunk.getFormat());
       myDialog.setTitle(getTitlePresentation(chunk.getSlicePresentation()));
+      boolean shouldSetColored = myTableCellRenderer == null || myTableCellRenderer.getColored();
```

Comment:
I probably miss something, but could you please explain what this condition for?